scipy/optimize/_highs/.github/workflows/build-windows.yml

```diff
@@ -0,0 +1,107 @@
+name: build-windows
```

Comment:
why all the CI files, `.gitignore`, etc.? should be removed it looks like

Reply:
Removed